docs/en/Xamarin.Essentials/FilePicker.xml

```diff
@@ -32,7 +32,7 @@
       </Parameters>
       <Docs>
         <param name="options">File picker options to use; may be null</param>
-        <summary>Starts file picker for selecting a single file.</summary>
+        <summary>Starts file picker for selecting files.</summary>
```

Comment:
I think you changed the wrong one here. The one down below seems the one that is actually wrong on line 63